data/data-pipeline/tox.ini

```diff
@@ -7,32 +7,17 @@ skip_missing_interpreters = true
 
 [testenv:lint]
 # lints python code in src and tests
-deps = -rrequirements.txt
-# These are "external" because they are dev dependencies that not in
-# "requirements.txt" and therefore not installed in this env, but they will be
-# available where this is being run. See:
-# https://stackoverflow.com/questions/47642747/tox-warningtest-command-found-but-not-installed-in-testenv
-allowlist_externals = black
-                      flake8
-                      pylint
 commands = black data_pipeline
            flake8 data_pipeline
-# Ignore tests this lint check because test dependencies are not installed here.
-           pylint data_pipeline --ignore tests
+           pylint data_pipeline
 
 [testenv:checkdeps]
 # checks the dependencies for security vulnerabilities and open source licenses
-deps = -rrequirements.txt
-# These are "external" because they are dev dependencies that not in
-# "requirements.txt" and therefore not installed in this env, but they will be
-# available where this is being run. See:
-# https://stackoverflow.com/questions/47642747/tox-warningtest-command-found-but-not-installed-in-testenv
-allowlist_externals = safety
-                      liccheck
+allowlist_externals = bash
 commands = safety check
-           liccheck
+           bash scripts/run-liccheck.sh
 
 [testenv:pytest]
 # Run tests
 deps = pytest
-commands = pytest
+commands = pytest --full-trace
```

Comment:
What is this flag?

Reply:
Making the output more verbose on failures: https://docs.pytest.org/en/latest/how-to/output.html#modifying-python-traceback-printing

I ran into https://github.com/usds/justice40-tool/issues/1108 while working on this, and the traceback helped.